rotkehlchen/history/events/structures/base.py

```diff
@@ -196,8 +205,28 @@ def _serialize_base_tuple_for_db(self) -> tuple[str, str, HISTORY_EVENT_DB_TUPLE
                 self.notes,
                 self.event_type.serialize(),
                 self.event_subtype.serialize(),
+                json.dumps(self.extra_data) if self.extra_data else None,
             ))
 
+    @staticmethod
+    def deserialize_extra_data(entry: tuple, extra_data: str | None) -> dict[str, Any] | None:
+        """Deserialize a history event's extra_data json from the db.
+        Args:
+            entry (tuple): event entry from the db to be logged on error
+            extra_data (str | None): extra data to be deserialized
+        Returns the extra data in a dict or None on error."""
+        if extra_data is None:
+            return None
+
+        try:
+            return json.loads(extra_data)
+        except json.JSONDecodeError as e:
+            log.debug(
```

Comment:
That's a pretty bad error. Put it as error.

```suggestion
            log.error(
```